horovod/spark/runner.py

```diff
@@ -307,6 +307,7 @@ def run_elastic(fn, args=(), kwargs={}, num_proc=None, min_np=None, max_np=None,
         elastic_timeout: Timeout for elastic initialisation after re-scaling the cluster.
                        If not set, falls back to `HOROVOD_ELASTIC_TIMEOUT` environment variable value.
                        If it is not set as well, defaults to 600 seconds.
+        reset_limit: Maximum number of resets after which the job is terminated.
```

Comment:
`Maximum number of resets before the job is terminated.`, 'after' sounds like right after the N-th reset the job is terminated, so we have N-1 retries / resets.

Reply:
In reality, there is a reset at the beginning when the job is started, but that is an implementation detail.  From the user perspective, we consider a reset to happen when a worker is added / removed after the initial registration.  So if I set `--reset-limit 0`, for example, then my job can never resize.  If I set it to `1`, it can resize exactly once, etc.  I'll update the docs for this param.